words/core.asm

```diff
@@ -763,7 +763,7 @@ z_bracket_tick: rts
                 ; """
 xt_buffer_colon:
 w_buffer_colon:
-                jsr w_create
+                jsr w_create            ; will show PFA size 2 in SEE
```

Comment:
This is the only word we don't have an accurate PFA size for, it'll just show the default 2 of dovar.